### PR TITLE
Fix issues reported on expresson syntax doc

### PR DESCRIPTION
**Pull request by shaniR**
## PurposeThis PR fixes the following issues.

closes wso2-enterprise/choreo#3860
closes wso2-enterprise/choreo#3858
closes wso2-enterprise/choreo#3859
closes wso2-enterprise/choreo#3857
closes wso2-enterprise/choreo#3856
closes wso2-enterprise/choreo#3855
closes wso2-enterprise/choreo#3854
closes wso2-enterprise/choreo#3853
closes wso2-enterprise/choreo#3828
closes wso2-enterprise/choreo#3827
closes wso2-enterprise/choreo#3992

### CODE REVIEW

File: en/docs/references/ballerina-expression-syntax.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+  1234
+  ```
+  ```ballerina
+  -10
+  ```
+  ```ballerina
+  +2020
+  ```
+
+### Decimal Floating-Point Number
```
Comment (anupama-pathirage) [inline en/docs/references/ballerina-expression-syntax.md:39]: Do we need to say `Number` here? As for the previous ones we simply use the type name (i.e. `Signed Integer` and `String` : Not `Signed Integer Number`)

File: en/docs/references/ballerina-expression-syntax.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+  -8593.992
+  ```
+  ```ballerina
+  +950.930
+  ```
+  ```ballerina
+  0
+  ```
+
+### Boolean Value
```
Comment (anupama-pathirage) [inline en/docs/references/ballerina-expression-syntax.md:61]: Same as above. Do we need to say `Value` here?

File: en/docs/references/ballerina-expression-syntax.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+   firstNumber == 100
+  ```
+  
+- Checking if the values of two variables (`firstNumber` and `secondNumber`) are equal. The following returns true when the value of the two variables is equal.
+
+  ```ballerina
+  firstNumber == secondNumber
+  ```
+  
+- Checking if the values of two variables (`firstNumber` and `secondNumber`) are NOT equal. The following returns true when the value of the two variables are different.
```
Comment (anupama-pathirage) [inline en/docs/references/ballerina-expression-syntax.md:286]: Do we need to use uppercase `NOT` here?

File: en/docs/references/ballerina-expression-syntax.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+
+  ```ballerina
+  firstNumber != secondNumber
+  ```
+
+### Checking the Type of Variable
+
+The `is` operator is used to assert the type of a variable.
+
+#### Examples
```
Comment (anupama-pathirage) [inline en/docs/references/ballerina-expression-syntax.md:296]:
```suggestion
#### Example
```

File: en/docs/references/ballerina-expression-syntax.md (hunk 1 above)
Comment (shaniR) [reply en/docs/references/ballerina-expression-syntax.md:39]: No I think, when you say DFP, it's generally used together with another word to emphasize if it's the number you are talking about or the operation. But when you say signed integers it is clear to the reader that it refers to the number.  Hence, used the word 'number'  in DFP to give more clarity to the reader.

File: en/docs/references/ballerina-expression-syntax.md (hunk 2 above)
Comment (shaniR) [reply en/docs/references/ballerina-expression-syntax.md:61]: yeah we can remove that.

File: en/docs/references/ballerina-expression-syntax.md (hunk 3 above)
Comment (shaniR) [reply en/docs/references/ballerina-expression-syntax.md:286]: used it for emphasis.

File: en/docs/references/ballerina-expression-syntax.md (hunk 4 above)
Comment (shaniR) [reply en/docs/references/ballerina-expression-syntax.md:296]: corrected.

File: en/docs/references/ballerina-expression-syntax.md (hunk 1 above)
Comment (anupama-pathirage) [reply en/docs/references/ballerina-expression-syntax.md:39]: Ack

**Comment by anupama-pathirage** [comment]
LGTM!

File: en/docs/references/ballerina-expression-syntax.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+  
+- Calculating the modulus of two numerical variables (`firstNumber` and `secondNumber`).
+
+  ```ballerina
+  firstNumber % secondNumber
+  ```
+
+### Checking The Equality of Basic Types 
+
+In the [Ballerina language](https://ballerina.io/), basic types are `string`, `int`, `float`, `decimal`, and `boolean`. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Note that the resulting value from these operators is always of `boolean` type.
```
Comment (anupama-pathirage) [inline en/docs/references/ballerina-expression-syntax.md:258]: It seems we need `()` too here

Comment (shaniR) [reply en/docs/references/ballerina-expression-syntax.md]: done

**Pull request merged**